tests/cupyx_tests/scipy_tests/linalg_tests/test_matfuncs.py

```diff
@@ -103,3 +103,8 @@ def test_2x2_input(self, xp, scp):
     def test_nx2x2_input(self, xp, scp, a):
         a = xp.asarray(a)
         return scp.linalg.expm(a)
+
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_complex(self, xp, scp):
```

Comment:
I am a bit confused, shouldn't we just have one test marked with `@testing.for_all_dtypes()` and we'd cover all spectrum? Seems like we don't have good dtype test coverage for `expm` even after this PR (ex: float16float32/float64/int/...)

Reply:
Fair enough. Updated to the suggestion.